packages/replay/src/types.ts

```diff
@@ -22,6 +22,11 @@ export interface Options {
   apiKey?: string;
   verbose?: boolean;
   agent?: any;
+
+  /**
+   * Fail the recording upload if any part of the upload fails.
+   */
+  strict?: boolean;
```

Comment:
Are there situations where we wouldn't want this behavior? I guess maybe people wouldn't appreciate their CI failing if the recording upload fails?

Reply:
I have no idea why you wouldn't want this behaviour, but this is public/in the wild, and so making this anything other than opt-in could break users downstream, thus I'm adding the flag.